dev/com.ibm.ws.anno/src/com/ibm/ws/annocache/targets/cache/internal/TargetCacheImpl_ReaderBinary.java

```diff
@@ -218,6 +219,11 @@ public String requireCompact() throws IOException {
     //
 
     public void requireHeader(String tableName, String tableVersion) throws IOException {
```

Comment:
Is this method needed any longer?

Reply:
I was asking about the `requireHeader` method, not the `requireCompact()` method.

Reply:
You cleaned up the method so I marked it resolved.